src/views/Dashboard.tsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
 import { Container } from '../components/Container/Container';
+import { LeadsList } from 'components/LeadsList/LeadsList';
+import { useAppSelector } from 'store/hooks';
+import { selectLeads } from 'data/leads/slice';
+import { formatLeadsArray } from 'utils/formatLeadsArray/formatLeadsArray';
 
 import styles from './Dashboard.module.scss';
 
 export const Dashboard = () => {
+   const leads = useAppSelector(selectLeads);
```

Comment:
**nitpick:** As we use hooks, you don't have to pass data here - just fetch it within the component that uses it